test/src/integration/integration_scalardl_test.go

```diff
@@ -115,3 +122,26 @@ func getExternalIP(t *testing.T) string {
 
 	return output
 }
+
+func checkTCPConnect(t *testing.T, host string) string {
+	logger.Logf(t, "Check tcp connection: %s", host)
+
+	timeout := 10 * 60
+	status := "NG"
+
+	for i := 0; i <= timeout; i += 10 {
```

Comment:
I think it's a bit weird to see using `i` as a time.
It can be simply a counter to do the same thing.
```suggestion
	for i := 0; i <= numRetries; i++ {
```

Reply:
OK! Refactered in 5d86233.